.github/workflows/main.yml

```
@@ -79,6 +83,12 @@ jobs:
         sudo apt-get update
         sudo apt-get install -y gcc-aarch64-linux-gnu qemu-user
 
+    - name: Install S390X toolchain and qemu
```

```suggestion
    - name: Install s390x toolchain and qemu
```
for consistency with the other references to this architecture.